components/modal/frame.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, compose, createRef } from '@wordpress/element';
+import { keycodes } from '@wordpress/utils';
```

Comment:
`keycodes` was updated to be its own package in #7577 . I'm not sure why this isn't logging any deprecation warnings, but it should be updated.

cc @tofumatt

Reply:
Addressed.